README.md

```diff
@@ -24,6 +24,16 @@ def deps do
 end
 ```
 
+## Basic Usage
+
+``` elixir
+iex(1)> Jason.encode!(%{"age" => 44, "name" => "Steve Irwin", "nationality" => "Australian"})
+"{\"age\":44,\"name\":\"Steve Irwin\",\"nationality\":\"Australian\"}"
+
+iex(2)> Jason.decode!("{\"age\":44,\"name\":\"Steve Irwin\",\"nationality\":\"Australian\"}")
```

Comment:
I wonder instead of escaping the quotes in the string, what about using another quote type for the string like:

```elixir
iex(2)> Jason.decode!("{'age':44,'name':'Steve Irwin','nationality':'Australian'}")
```

Or:

```elixir
iex(2)> Jason.decode!(~S({"age":44,"name":"Steve Irwin","nationality":"Australian"}))
```

All the backslashes are noisy...  Although it does match the printed output of the prior `encode!` call...